ops/src/purge/errored_txs_purger.rs

```diff
@@ -0,0 +1,296 @@
+use anyhow::Result;
```

Comment:
I am fine with this code being part of the current purger file for now. modules in the future are something like

```
/src/purge/
  queues/
    paginate.rs
    mark_deletion.rs
    purge.rs
    mod.rs
  cnft.rs
  mint.rs
  token.rs
  mod.rs 
```
but I don't think we need this yet.